### DISCUSSION TITLE
Welcome to djs-ticket-system Discussions!

### DISCUSSION THREAD

**Discussion by LunarTaku** [LunarTaku/djs-ticket-system#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by MonishKrishna2009** [comment]
Hey it's just a suggestion that you can use modals for the setup command you know it would look cool

**Reply by LunarTaku** [reply]
i could im going to try and do that, thanks for the suggestion!
